testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -183,7 +188,19 @@ def test_rmsd_single_frame(self):
 
     def test_mass_weighted_and_save(self):
         RMSD = MDAnalysis.analysis.rms.RMSD(self.universe, select='name CA',
-                                            step=49, mass_weighted=True).run()
+                                            step=49, weights='mass').run()
+        RMSD.save(self.outfile)
+        saved = np.loadtxt(self.outfile)
+        assert_array_almost_equal(RMSD.rmsd, saved, 4,
+                                  err_msg="error: rmsd profile should match " +
+                                          "test values")
+
+    def test_mass_weighted_and_save_deprecated(self):
+        with warnings.catch_warnings(record=True) as warn:
+            warnings.simplefilter('always')
+            RMSD = MDAnalysis.analysis.rms.RMSD(self.universe, select='name CA',
+                                                step=49, mass_weighted=True).run()
+        assert(len(warn), 1)
```

Comment:
Travis CI build seems to whine about this with `testsuite/MDAnalysisTests/analysis/test_rms.py:203: [W0199(assert-on-tuple), TestRMSD.test_mass_weighted_and_save_deprecated] Assert called on a 2-uple. Did you mean 'assert x,y'?`